wordpress-mobile/gutenberg-mobile
# Update Aztec reference.

This PR updates RNAztec reference to test [this PR](https://github.com/wordpress-mobile/react-native-aztec/pull/70) related to cocoapods integration improvements.

To test:

- Checkout the gutenberg-mobile PR. (soon to be created)
- Check that the events are passed and handled correctly (bold, italic etc...)
